web/viewer.css

```diff
@@ -1010,6 +1010,10 @@ html[dir="rtl"] .secondaryToolbarButton > span {
   content: url(images/secondaryToolbarButton-rotateCw.png);
 }
 
+.secondaryToolbarButton.selectTool::before {
+  content: url(images/secondaryToolbarButton-selectTool.png);
```

Comment:
> You could try asking on IRC. I believe he (and other designers) are usually in the #ux channel. Otherwise, I'll check if I can create the icons.

It seems that the #ux channel is pretty much dead, and that the UX teams uses other means of communication now :-(
I've unfortunately not received any response from Stephen Horlander to the email I sent (last year); and it'd be nice if we're able to move forward with this PR.

@timvandermeij If you have time, do you think that you'd be able to create the icons[1]?
(As previously mentioned, they would *obviously* land in a separate commit with you as the author.)

---
[1] I actually tried to come up with something myself here, but since I'm rubbish at design, it didn't look good enough to even consider including in PDF.js :-)